patches/ui-views-controls-menu-menu_config.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/ui/views/controls/menu/menu_config.cc b/ui/views/controls/menu/menu_config.cc
+index aab2666f7a546ce80446e6d4e83efd6b2b355581..16f004ae674cdfdf442dcc2a012409b013734d5d 100644
+--- a/ui/views/controls/menu/menu_config.cc
++++ b/ui/views/controls/menu/menu_config.cc
+@@ -71,6 +71,7 @@ void MenuConfig::InitCR2023() {
+ // static
+ const MenuConfig& MenuConfig::instance() {
```

Comment:
I think it's possible to redefine this as `#define instance() instance_ChromiumImpl()` here and in the header and implement a custom `instance()` method.

Reply:
Done. Thanks!